readme.txt

```diff
@@ -309,8 +309,7 @@ The plugin is made with love by [Modern Tribe Inc](http://m.tri.be/2s).
 == Changelog ==
 
 = [4.6.10.1] 2018-01-25 =
-
-
+* Fix - Make sure rewrite rule for `/events` is not hardcoded and is based on dynamic option field [98463]
```

Comment:
Especially with a hotfix, it's a nice touch to acknowledge folks who highlighted the problem. Maybe add: _(thanks to elecevan, nickfetner and everyone else who reported this problem)_

Reply:
For sure, will update this one.